2016-06-28-Rust-Reach-Fractal.md

```diff
@@ -0,0 +1,91 @@
+# Where are we going with Tessel? Rust, Reach, and Fractal
+
+The aim of the Tessel Project is to create a fully open source hardware & software platform that makes it easy and intuitive to develop Internet-connected devices.
+
+With Tessel 1, we tackled the "easy and intuitive" part of that mission statement. Tessel 2 furthered that piece of the mission, and began to work towards letting developers build at scale. Specifically, I think we've done quite well towards fulfilling two of our core philosophies:
+
+> **Developer experience is paramount.** Tessel should be the fastest way to build an idea, regardless of your background (or lack thereof) in electrical engineering or computer programming. Branding, documentation, and engineering design decisions should always take this into account.
+
+> **Device design should focus on user experience rather than on implementation.** Tessel uses high level languages and modular plug & play hardware (see philosophy here) so that device creators can fully prototype and test device functionality before worrying about optimization and implementation specifics.
+
+Our next efforts should take us down the road to this less-fulfilled core belief:
+
+> **Tessel should be practical to use.** Tessel as a platform cannot have a big impact unless it is cost effective, reliable, and available for purchase. *It needs to be possible to purchase Tessel hardware at affordable prices, from quantity one up to production-level quantities (10,000+).*
+
+(Emphasis added.)
+
+## Where we are: the vision of Tessel 2
+
+When we began to design Tessel 2, we envisioned it as the basis of products that our users could take to market. At the core of its design was the key idea:
+
+**People should be able to scale the tools they use for prototyping all the way into production.**
+
+This is a radical idea.
+
+If you're a Node developer, this prototyping --> production path should feel familiar. Both Node.JS and Tessel subscribe to the philosophy that you should be able to build modularly. Both let you design iteratively, and swap out packaged components according to your (possibly changing) needs.
+
+In hardware, this is nearly unheard of. It is very common to build a first prototype to show technical viability, then start from scratch to design a PCB that works with parts you can get at the x1000 scale.
+
+With Tessel 2, we say: you shouldn't have to. This concept drove the following design decisions:
+
+* All of the parts used for Tessel 2 are relatively new (they will continue to be manufactured and supported for a few years) and can be purchased in low (x1000) quantities.
```

Comment:
The MediaTek has been on the market for many years but is still easily sourceable. You may want to remove "relatively new".